sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -41,9 +41,7 @@
 #include "fmt.h"
 #include "log.h"
 #include "sched.h"
-#if (CONFIG_GNRC_NETIF_MIN_WAIT_AFTER_SEND_US > 0U)
```

Comment:
I think there was a problem with an all-out include, that's why I `#if`'d it. Maybe add `|| IS_USED(MODULE_GNRC_NETIF_BUS)`?

Reply:
Not sure why a header file would cause trouble - Murdock seems fine with it (except for insufficient memory boards for the new test).

I would try to avoid `#ifdefs` where they are not needed to not clutter the code

Reply:
Ok, if murdock is happy NVM.

Reply:
> Ok, if murdock is happy NVM.

Ad https://github.com/RIOT-OS/RIOT/commit/9db92f0f81921229c28b2e57b2af919a5212f6d4, apparently it was not?

Reply:
Yes turns out the include causes problems if the board does not support xtimer (IMHO that should be fixed in the header file)